packages/lesswrong/components/ea-forum/EASurveyBanner.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useCallback } from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import { TypeformPopupEmbed } from "../common/TypeformEmbeds";
+import { useTracking } from "../../lib/analyticsEvents";
+import { useCookiesWithConsent } from "../hooks/useCookiesWithConsent";
+import { HIDE_EA_FORUM_SURVEY_BANNER_COOKIE } from "../../lib/cookies/cookies";
+import moment from "moment";
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    position: "sticky",
+    top: 0,
+    zIndex: 10000, // The typeform popup has z-index 10001
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    textAlign: "center",
+    gap: "25px",
+    padding: 12,
+    background: theme.palette.primary.main,
```

Comment:
```suggestion
    background: theme.palette.buttons.alwaysPrimary,
```

Since the text is always white, we should prob use the darker primary even in dark mode